common/xbps-src/shutils/common.sh

```diff
@@ -597,4 +579,12 @@ setup_pkg() {
     fi
 
     source_file $XBPS_COMMONDIR/environment/build-style/${build_style}.sh
+
+    # Source all build-env files that are defined
+    for f in $build_env; do
+        if [ ! -r $XBPS_BUILDENVDIR/${f}.sh ];  then
+            msg_error "$pkgver: cannot find build environment $XBPS_BUILDENVDIR/${f}.sh!\n"
```

Comment:
$XBPS_BUILDENVDIR/${f}.sh!\n 

I would change to:
${XBPS_BUILDENVDIR}/${f}.sh!\n

Brackets I learned in Arch's AUR can avoid some issues with variables

Reply:
There isn't a reason here, bracket expansion it useful in cases like:

```
export CARGO_TARGET_${RUST_TARGET}_LINKER="$CC"
```

otherwise

```
export CARGO_TARGET_$RUST_TARGET_LINKER="$CC"
```

will produce wrong results